eth/contracts/StableSwap/StableSwapTwoPool.sol

```diff
@@ -96,19 +97,33 @@ contract StableSwapTwoPool is Ownable, ReentrancyGuard {
         uint256[N_COINS] fees,
         uint256 token_supply
     );
-    event RemoveLiquidityOne(address indexed provider, uint256 index, uint256 token_amount, uint256 coin_amount);
+    event RemoveLiquidityOne(
+        address indexed provider,
+        uint256 index,
+        uint256 token_amount,
+        uint256 coin_amount
+    );
     event RemoveLiquidityImbalance(
         address indexed provider,
         uint256[N_COINS] token_amounts,
         uint256[N_COINS] fees,
         uint256 invariant,
         uint256 token_supply
     );
-    event CommitNewFee(uint256 indexed deadline, uint256 fee, uint256 admin_fee);
+    event CommitNewFee(
+        uint256 indexed deadline,
+        uint256 fee,
+        uint256 admin_fee
+    );
     event NewFee(uint256 fee, uint256 admin_fee);
-    event RampA(uint256 old_A, uint256 new_A, uint256 initial_time, uint256 future_time);
+    event RampA(
+        uint256 old_A,
+        uint256 new_A,
+        uint256 initial_time,
+        uint256 future_time
+    );
     event StopRampA(uint256 A, uint256 t);
-    event SetBNBGas(uint256 bnb_gas);
```

Comment:
Btw is this a global variable `bnb_gas` or local parameter to an event?

Reply:
Its like a fn signature, so just named variable